stress/src/throughput.rs

```diff
@@ -26,7 +26,7 @@ where
         STOP.store(true, Ordering::SeqCst);
     })
     .expect("Error setting Ctrl-C handler");
-    let num_threads = num_cpus::get_physical();
+    let num_threads = num_cpus::get();
```

Comment:
feel like the same advice I constantly give to my little kids! No physical fights 🤣